.travis.yml

```diff
@@ -122,13 +120,39 @@ install:
 script:
   - ./bootstrap
   - ./configure $FFCONFIG
-  - make
-  - make install
+  - make -j$(nproc)
+  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then mkdir -p appdir ; make install DESTDIR=$(readlink -f appdir); fi
+  - if [ "$TRAVIS_OS_NAME" != "linux" ]; then make install; fi
   - make check
   - if [ ! -z "$LINUX_FULL" ]; then make distcheck; fi
   - fontforge -version
   - $PYTHON -c "import fontforge; import psMat; print(fontforge.__version__, fontforge.version()); fontforge.font();"
   - if [ "$TRAVIS_OS_NAME" == "osx" ] && [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" == "master" ]; then travis-scripts/ffosxbuild.sh $PREFIX $TRAVIS_COMMIT; fi
+  - |
```

Comment:
Move all of this into a packaging script, similar to `travis-scripts/ffosxbuild.sh`

Reply:
Moved there. Personally I find it easier if everything is done directly in `.travis.yml` but it's a matter of personal preference.